packages/terra-arrange/src/Arrange.scss

```diff
@@ -12,16 +12,15 @@
   // Makes sure fill expands to fill the remaining space of its parent container
   // Display set to ensure correct appreance if applied to inline elements like span
```

Comment:
This comment about display can be removed.

Reply:
removed the comment